_source/logzio_collections/_metrics-sources/azure-monitor.md

```diff
@@ -0,0 +1,179 @@
+```
+
+To complete the process, sign in through your browser.
+
+##### Get your subscription details
+
+Run this command:
+
+```shell
+az account show | grep '"id"\|"tenantId"'
+```
+
+The response shows the subscription ID (the `id` field)
+and tenant ID (the `tenantId` field).
+You'll need this information later on, so paste it in your text editor.
+
+###### Sample response
+
+```text
+"id": "d94b1fba-0289-557e-b063-0b6bfc1bdca0",
+"tenantId": "9ae0715a-0689-56e8-bb88-2b22f1fa7299",
+```
+
+##### Create a new Azure AD application
+
+Run this command to create an AD application with reader permissions.
+
+Replace `<<SUBSCRIPTION-ID>>` with the `id` value from step 2.
+
+```shell
+az ad sp create-for-rbac --role reader \
+--scopes /subscriptions/<<SUBSCRIPTION-ID>> \
+-n logzio-metricbeat \
+| grep '"appId"\|"password"'
+```
+
+The response shows the client ID (the `appId` field)
+and client secret (the `password` field).
+You'll need this information later on, so paste it in your text editor.
+
+###### Sample response
+
+```text
+"appId": "3dcdf53e-f93f-5902-8df2-235c8635aa4d",
+"password": "e6ab6d24-4907-5d11-a132-a171ef55355d",
+```
+
+##### Download the Logz.io certificate
+
+Now we'll set up your Metricbeat server.
+
+Download the Logz.io public certificate to your certificate authority folder.
+
+```shell
+sudo wget https://raw.githubusercontent.com/logzio/public-certificates/master/COMODORSADomainValidationSecureServerCA.crt -P /etc/pki/tls/certs/
+```
+
+##### _(Optional)_ Disable the system module
+
+By default, Metricbeat ships system metrics from its host.
+Disable this module so you don't unintentionally send host metrics:
+
+```shell
+sudo metricbeat modules disable system
+```
+
+##### Add Logz.io configuration
+
+Now you'll set up the Metricbeat
+to collect metrics from Azure Monitor.
+
+You'll need to replace the values surrounded by angle brackets
+`<< >>`
+using the parameters below the code block. 👇
+
+```yaml
+metricbeat.config.modules:
+  path: ${path.config}/modules.d/*.yml
+  reload.enabled: false
+
+metricbeat.modules:
+- module: azure
+  metricsets:
+  - monitor
+  enabled: true
+  period: 300s
+  client_id: '${AZURE_CLIENT_ID:"<<CLIENT-ID>>"}' # `appId` from step 3
+  client_secret: '${AZURE_CLIENT_SECRET:"<<CLIENT-SECRET>>"}' # `password` from step 3
+  tenant_id: '${AZURE_TENANT_ID:"<<TENANT-ID>>"}' # `tenantId` from step 2
+  subscription_id: '${AZURE_SUBSCRIPTION_ID:"<<SUBSCRIPTION-ID>>"}' # `id` from step 2
+  resources:
+    - resource_query: "resourceType eq '<<RESOURCE-TYPE>>'"
+      metrics:
+      - name: ["*"]
+        namespace: "<<RESOURCE-TYPE>>"
+
+fields:
+  logzio_codec: json
+  token: <<SHIPPING-TOKEN>>
+fields_under_root: true
+ignore_older: 3hr
+type: metrics
+output.logstash:
+  hosts: ["<<LISTENER-HOST>>:5015"]
+  ssl.certificate_authorities: ['/etc/pki/tls/certs/COMODORSADomainValidationSecureServerCA.crt']
+```
+
+To send storage metrics, add this code block to the `metric.modules` object:
```

Comment:
I think we need to explain that for some of the services metricbeat has a pre-defined module (and add a link to the list of modules) and only then give storage as an example. I wasn't sure how storage is related to the flow of reading this.

Reply:
That's beyond the scope of this doc, I think. Your suggestion definitely makes sense for a Metricbeat reference doc, which we should add to the shippers tab

Reply:
tomorrow there will be more modules...

Reply:
It's not a comprehensive documentation on Metricbeat and its modules, though. It's a step-by-step on getting metrics from Monitor using metricbeat.